mmdet/datasets/pipelines/transforms.py

```diff
@@ -269,7 +269,7 @@ def _resize_seg(self, results):
                     results['scale'],
                     interpolation='nearest',
                     backend=self.backend)
-            results['gt_semantic_seg'] = gt_seg
```

Comment:
I deleted this line. The `seg_fileds` only has one key now, which is `gt_semantic_seg`. So deleting it is safe.